src/js/program/program-controller.js

```diff
@@ -266,7 +266,11 @@ class ProgramController extends Events {
             return;
         }
 
-        backgroundMediaController.mediaModel.attributes.mediaState = 'buffering';
+        let mediaState = backgroundMediaController.mediaModel.attributes.mediaState;
+        if (mediaState !== 'paused') {
+            mediaState = 'buffering';
```

Comment:
Setting `mediaState` to buffering here does nothing. You have to update `backgroundMediaController.mediaModel.attributes.mediaState`.

Reply:
So setting `mediaState = backgroundMediaController.mediaModel.attributes.mediaState` and then setting `mediaState` to something else doesn't work?

Reply:
it's a variable.